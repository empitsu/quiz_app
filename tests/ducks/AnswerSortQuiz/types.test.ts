import {
  POP_FROM_SELECTED_OPTIONS,
  PUSH_T0_SELECTED_OPTIONS,
  RESET,
} from "../../../src/ducks/AnswerSortQuiz/types";

describe("types of AnswerSortQuiz", () => {
  describe("POP_FROM_SELECTED_OPTIONS", () => {
    it("should be a action type of popFromSelectedOptions", () => {
      expect(POP_FROM_SELECTED_OPTIONS).toMatchSnapshot();
    });
  });
  describe("PUSH_T0_SELECTED_OPTIONS", () => {
    it("should be a action type of pushToSelectedOptions", () => {
      expect(PUSH_T0_SELECTED_OPTIONS).toMatchSnapshot();
    });
  });
  describe("RESET", () => {
    it("should be a action type of reset", () => {
      expect(RESET).toMatchSnapshot();
    });
  });
});
