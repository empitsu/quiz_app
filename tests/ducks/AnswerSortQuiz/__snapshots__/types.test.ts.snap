// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`types of AnswerSortQuiz POP_FROM_SELECTED_OPTIONS should be a action type of popFromSelectedOptions 1`] = `"POP_FROM_SELECTED_OPTIONS"`;

exports[`types of AnswerSortQuiz PUSH_T0_SELECTED_OPTIONS should be a action type of pushToSelectedOptions 1`] = `"PUSH_T0_SELECTED_OPTIONS"`;

exports[`types of AnswerSortQuiz RESET should be a action type of reset 1`] = `"RESET"`;
